src/containers/DailyReportPage/DailyReportContainer.js

```diff
@@ -13,84 +12,146 @@ import {
   ReportsRowColumn,
   IconBtn
 } from 'components/Shared/Reports/styles';
+import gql from 'graphql-tag';
+import { Query, graphql, compose } from 'react-apollo';
 import history from '../../utils/history';
 import { ContentWrapper } from '../../styles/App';
-import { headerItems } from './constants';
+import { headerItems, itemsAmount } from './constants';
 import formatDate from '../../utils/formatDate';
 
 const DAILY_REPORTS_QUERY = gql`
-  query DAILY_REPORTS_QUERY {
-    userReports {
-      id
-      emotion
-      title
-      achievement
-      plan
-      comment
-      createdAt
+  query DAILY_REPORTS_QUERY($first: Int, $skip: Int) {
+    userReports(first: $first, skip: $skip) {
+      count
+      dailyReports {
+        id
+        emotion
+        title
+        achievement
+        plan
+        comment
+        createdAt
+      }
     }
   }
 `;
 
-const DailyReportContainer = ({ match }) => (
-  <Query query={DAILY_REPORTS_QUERY}>
-    {({ data, error, loading }) => {
-      { loading && <p>Loading...</p> }
-      { error && <p>Error: {error.message}</p> }
-      if (isEmpty(data.userReports)) {
-        return (
-          <ContentWrapper>
-            <h2>No Daily Report is submitted yet!</h2>
-          </ContentWrapper>
-        );
-      }
+export const USER_DAILY_REPORTS_COUNT_QUERY = gql`
+  query USER_DAILY_REPORTS_COUNT_QUERY {
+    userDailyReportsCount @client {
```

Comment:
Query & Mutation with `@client` only interact with the cache and never get data from server side.
https://www.apollographql.com/docs/link/links/state.html